application/F3DOptionsTools.cxx

```diff
@@ -74,7 +74,8 @@ static inline const std::array<CLIGroup, 8> CLIOptions = {{
       { "watch", "", "Watch current file and automatically reload it whenever it is modified on disk", "<bool>", "1" },
       { "load-plugins", "", "List of plugins to load separated with a comma", "<paths or names>", "" },
       { "scan-plugins", "", "Scan standard directories for plugins and display available plugins (result can be incomplete)", "", "" },
-      { "screenshot-filename", "", "Screenshot filename", "<filename>", "" } } },
+      { "screenshot-filename", "", "Screenshot filename", "<filename>", "" } ,
+      { "keep-camera-between-file", "", "Keep the camera state when loading a new file group.", "<bool>", "1" } } },
   { "General",
```

Comment:
lets not add a CLI option for this

Reply:
> lets not add a CLI option for this

Do you think it's worth keeping the it as a json config option? If so, should the entry in `DefaultAppOptions` be kept? (The comment says it is for ` The "App" CLI options`; I am wondering if there is a list of configurable non-CLI options)

Reply:
> > lets not add a CLI option for this
> 
> Do you think it's worth keeping it as a json config option? If so, should the entry in `DefaultAppOptions` be kept? (The comment says it is for ` The "App" CLI options`; I am wondering if there is a list of configurable non-CLI options)

Upon a second thought, I don't think we should add a json config option either; if the user wants keeping camera state as the default behavior, they can remap the Left/Right keys in their configuration.

Reply:
its not an option at all, just two different bindings